source/agora/consensus/protocol/Nominator.d

```diff
@@ -1014,6 +1014,13 @@ extern(D):
         this.network.gossipBlockSignature(block_sig);
     }
 
+    /// Were we active in this block (i.e. we are not catching up after being offline)
```

Comment:
Even if were offline during the nomination, Ledger will only accept a single block at a height. So we can't really sign multiple blocks at the same height, am I missing something?

Reply:
It was an unlikely edge case although if it happened would be disastrous to the owner of the validator stake. See https://github.com/bosagora/agora/issues/2299 for the background.